superset-frontend/src/views/CRUD/dashboard/DashboardList.tsx

```diff
@@ -337,6 +350,26 @@ function DashboardList(props: DashboardListProps) {
       ),
       paginate: true,
     },
+    {
+      Header: t('Created By'),
+      id: 'created_by',
+      input: 'select',
+      operator: 'rel_o_m',
+      unfilteredLabel: 'All',
+      fetchSelects: createFetchRelated(
+        'dashboard',
+        'created_by',
+        createErrorHandler(errMsg =>
+          props.addDangerToast(
+            t(
+              'An error occurred while fetching chart dataset values: %s',
```

Comment:
```suggestion
              'An error occurred while fetching dashboard created by values: %s',
```

Reply:
can you also change the error message in the "owners" filter to say `dashboard` instead of `chart`